openapi/components/schemas/Invoices/TaxTypes/rebilly-avalara.yaml

```
@@ -0,0 +1,16 @@
+allOf:
+  - $ref: ../InvoiceTax.yaml
+  - type: object
+    required:
+      - calculator
+    properties:
+      calculator:
+        description: Type of tax calculator.
+        type: string
+      items:
+        type: array
+        description: Taxes that are applied to this invoice and calculated based on the related product `taxCategoryId`.
```

nitpick: for consistency with the description in `/schemas/Invoices/TaxTypes/rebilly-taxjar.yaml`
```suggestion
        description: Taxes that are applied to this invoice and calculated based on the `taxCategoryId` of the related product.
```